### ISSUE TITLE
GameUI.java throws a java.lang.IllegalArgumentException during runtime. 

### ISSUE THREAD

**Issue by Jalaj-Maheshwari** [udacity/ud282#2]
Hi,

I have found that during runtime, the GameUI.java throws a java.lang.IllegalArgumentException at line 49 with message "input==null!". 
**The following are the logs for same:**
Exception in thread "main" java.lang.IllegalArgumentException: input == null!
	at javax.imageio.ImageIO.read(ImageIO.java:1348)
	at com.udacity.GameUI.<init>(GameUI.java:49)
	at com.udacity.Game.main(Game.java:163)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)

Process finished with exit code 1

While tracing the issue, it seems that there exists some problem with the way grid.png, x.png & o.png images are read using ImageIO.read() function.

I have made some minor changes in code to fix the issue. If the maintainers feel that this is a genuine issue, I can proceed with a pull request for same. Thank you!

**Comment by PoBlue** [comment]
hello，i have same issue... 

How did you solve it?

Thx~

**Comment by Jalaj-Maheshwari** [comment]
Hi, 
I made the following 2 changes:

1. Moved the resources directory present under the src/main to same level as that of src ie. Project 1 - TicTacToe / resources.

2. Modified the way images such as grid.png, x.png & y.png were loaded by replacing the lines 39 to 41 
grid = ImageIO.read(classLoader.getResourceAsStream("grid.png"));
x = ImageIO.read(classLoader.getResourceAsStream("x.png"));
o = ImageIO.read(classLoader.getResourceAsStream("o.png"));

in GameUI.java as per latest version of code on github to 
  
grid = ImageIO.read(new FileInputStream("resources/grid.png"));
x = ImageIO.read(new FileInputStream("resources/x.png"));
o = ImageIO.read(new FileInputStream("resources/o.png"));

Hope this helps! Thank you!
